edit-post/components/header/plugins-more-menu-group/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill, MenuGroup } from '@wordpress/components';
+import { PluginContext } from '@wordpress/plugins';
+import { __ } from '@wordpress/i18n';
+
+const { Fill, Slot } = createSlotFill( 'PluginsMoreMenuGroup' );
+
+const PluginsMoreMenuGroup = ( { children } ) => (
+	// Plugin's context needs to be forwarded to make it work properly with Dropdown.
+	<PluginContext.Consumer>
```

Comment:
Is this due to the React <16.3 context API, or just Slot/Fill in general?

Reply:
I think it's because Slot/Fill. `Dropdown` uses slot, and there is also another slot for `PluginsMoreMenuGroup` in the menu. At some point, they aren't part of the same tree in the virtual DOM. It works for `Sidebar` without any issues.

@aduth, any ideas? You know much better Slot & Fill API.